### DISCUSSION TITLE
Pas de sortie EJP

### DISCUSSION THREAD

**Discussion by cyrilus31** [JC144/EDF_Simulateur_Prix#85] [Q&A]
Bonjour,

Super projet pour comparer et faire le travail que ne font évidemment pas les fournisseurs d'énergie pour aider leur client à trouver le meilleur forfait.

Je n'ai pas les compétences pour creuser, mais il n'y a pas de sortie pour l'EJP.

**Comment by JC144** [comment]
Qu'entendez-vous par sortie ?

**Reply by cyrilus31** [reply]
J'ai un fichier .csv sur 6 mois de données. Votre comparateur me donne bien le coût pour tous les tarifs EDF sauf EJP (qui est bien activé dans le fichier index).

**Reply by JC144** [reply]
La puissance ne doit pas être disponible sur EJP, je regarderai ce soir mais vous pouvez déjà essayer avec du 9kVA.

**Reply by cyrilus31** [reply]
Effectivement c'était lié à la puissance que j'avais entré. Désolé de vous avoir fait perdre votre temps et encore merci.
